rpm/containerd.spec

```diff
@@ -30,18 +30,22 @@ AutoReq: no
 
 Name: containerd.io
 Provides: containerd
-# For some reason on rhel >= 8 if we "provide" runc then it makes this package unsearchable
+# On RHEL >= 8, runc is packaged by Red Hat so it won't be provided by this package.
```

Comment:
Would it be possible to generate runc rpm too, to override the Red Hat version?

cc @thaJeztah @kolyshkin WDYT?

Reply:
Why make a separate _runc_ package when there already exist a native one? Isn't it better to use the OS version when there is one?

Reply:
OS version is often outdated